src/ocean/io/select/selector/TimeoutSelectedKeysHandler.d

```diff
@@ -195,8 +197,9 @@ class TimeoutSelectedKeysHandler: SelectedKeysHandler
 
     ***************************************************************************/
 
-    extern (C) private static int cmpPtr ( bool searching ) ( in void* a_,
-        in void* b_ )
+    extern (C) private static int cmpPtr ( bool searching ) (
+        /* d1to2fix_inject: scope */ Const!(void*) a_,
```

Comment:
`in void* a_`?

Reply:
Can't mix `in` and `scope` because former was supposed to include the latter (but not anymore). Hm, speaking about that, @mathias-lang-sociomantic do you think this restriction can/should be removed now?

Reply:
Which restriction ? That `in` doesn't include `scope` ? I would love that, but don't think Walter will agree. Also, [they don't play along very well](https://issues.dlang.org/show_bug.cgi?id=17408) :(

Reply:
Other way around, prohibiting `in scope` combo.

Reply:
Should the bug be fixed ? Definitely. Just didn't have  the time (nor considered it important enough, given how DIP1000 is broken) to get around it.

Reply:
I wasn't even sure it is viewed as bug now. Was definitely intentional when `in` was still supposed to mean `const scope`.